sparkmagic/sparkmagic/utils/configuration.py

```diff
@@ -63,7 +64,7 @@ def base64_kernel_python3_credentials():
 
 @_with_override
 def kernel_scala_credentials():
-    return {u'username': u'', u'base64_password': u'', u'url': u'http://localhost:8998'}
+    return {u'username': u'', u'base64_password': u'', u'url': u'http://localhost:8998', u'auth_type': constants.NO_AUTH}
```

Comment:
We need the same for the other kernels too:

- kernel_r_credentials
- kernel_python3_credentials